app/script/util/TimeUtil.js

```diff
@@ -28,67 +28,73 @@ z.util.TimeUtil = {
     return Date.now() - timeOffset;
   },
 
+  durationUnits: [
+    {
+      plural: 'ephemeralUnitsYears',
+      singular: 'ephemeralUnitsYear',
+      unit: 'y',
+      value: 1000 * 60 * 60 * 24 * 365,
```

Comment:
@bennyn If we agreed on a new duration for a 365 in milliseconds we should use it everywhere. The validation in the handler still uses 365.25.